src/Servers/IIS/IIS/src/BadHttpRequestException.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace Microsoft.AspNetCore.Server.IIS
 {
+    ///<inheritdoc/>
```

Comment:
Does this work?

Reply:
The answer, kind of: https://github.com/dotnet/aspnetcore/issues/26434.

I'd rather not remove inheritdoc here though, as it's a deprecated type and should be fixed. We use inheritdoc in 500 files, and definitely more once we start writing more docs.